lib/compat/wordpress-6.3/script-loader.php

```diff
@@ -13,3 +13,71 @@
 remove_action( 'wp_body_open', 'wp_global_styles_render_svg_filters' );
 remove_action( 'in_admin_header', 'wp_global_styles_render_svg_filters' );
 
+/**
+ * Collect the block editor assets that need to be loaded into the editor's iframe.
+ *
+ * @since 6.0.0
+ * @access private
+ *
+ * @global string $pagenow The filename of the current screen.
```

Comment:
A note for later, when I finish the review: this needs updating if we are changing the `globals` in use by the function.